PWGHF/TableProducer/HFXToJpsiPiPiCandidateSelector.cxx

```diff
@@ -191,16 +193,21 @@ struct HFXToJpsiPiPiCandidateSelector {
       auto trackPos = hfCandX.index1_as<aod::BigTracksPID>(); //positive daughter
       auto trackNeg = hfCandX.index2_as<aod::BigTracksPID>(); //negative daughter
 
+      int selJpsiToEE = 1;
+      int selJpsiToMuMu = 1;
+
       // check if flagged as X --> Jpsi Pi Pi
-      if (!(hfCandX.hfflag() & 1 << XToJpsiPiPi)) {
-        hfSelXToJpsiPiPiCandidate(0);
-        // Printf("X candidate selection failed at hfflag check");
-        continue;
+      if (!(hfCandX.hfflag() & 1 << hf_cand_x::DecayType::XToJpsiToEEPiPi)) {
+        selJpsiToEE = 0;
+      }
+
+      if (!(hfCandX.hfflag() & 1 << hf_cand_x::DecayType::XToJpsiToMuMuPiPi)) {
+        selJpsiToMuMu = 0;
       }
 
       // daughter track validity selection
       if (!daughterSelection(trackPos) || !daughterSelection(trackNeg)) {
-        hfSelXToJpsiPiPiCandidate(0);
+        hfSelXToJpsiPiPiCandidate(0, 0);
         // Printf("X candidate selection failed at daughter selection");
```

Comment:
Is the function `daughterSelection` needed? The current version doesn't do anything.

Reply:
The current version do nothing but could be used in future to add further selections on the pions